server/ngwmn/services/ngwmn.py

```diff
@@ -440,16 +487,16 @@ def get_statistics(agency_cd, site_no):
     }
 
     overall = get_statistic(agency_cd, site_no, 'wl-overall')
-    if overall['is_fetched']:
+    if overall.get('is_fetched'):
         stats['overall'] = overall
         site_info = get_statistic(agency_cd, site_no, 'site-info')
         alt_datum_cd = ''
-        if site_info['is_fetched']:
+        if site_info.get('is_fetched'):
             alt_datum_cd = site_info['altdatumcd']
 
-        if overall['is_ranked']:
+        if overall.get('is_ranked'):
             monthly = get_statistic(agency_cd, site_no, 'wl-monthly')
-            if monthly['is_fetched']:
+            if monthly.get('is_fetched'):
```

Comment:
I'm not going to do this as part of this ticket but if I have down time this sprint, I may try refactoring some of the statistics code so that we are just fetching the data and not manipulating it in these functions. I talked to Dave U about it and he was fine with that.